x-pack/test/functional/es_archives/entity/risks/data.json

```diff
@@ -0,0 +1,159 @@
+{
+  "type": "doc",
+  "value": {
+    "index": "risk-score.risk-score-latest-default",
+    "id": "1",
+    "source": {
+      "host": {
+        "name": "suricata-zeek-sensor-toronto",
+        "risk": {
+          "calculated_score_norm": 96,
+          "calculated_level": "Critical"
+        }
+      },
+      "ingest_timestamp": "2022-08-15T16:32:16.142561766Z",
+      "@timestamp": "2022-08-12T14:45:36.171Z"
+    },
+    "type": "_doc"
+  }
+}
+
+{
+  "type": "doc",
+  "value": {
+    "id": "2",
+    "index": "risk-score.risk-score-latest-default",
+    "source": {
+      "host": {
+        "name": "suricata-sensor-london",
+        "risk": {
+          "calculated_score_norm": 20,
+          "calculated_level": "Low"
+        }
+      },
+      "ingest_timestamp": "2022-08-15T16:32:16.142561766Z",
+      "@timestamp": "2022-08-12T14:45:36.171Z"
+    },
+    "type": "_doc"
+  }
+}
+
+{
+  "type": "doc",
+  "value": {
+    "id": "3",
+    "index": "risk-score.risk-score-latest-default",
+    "source": {
+      "host": {
+        "name": "zeek-newyork-sha-aa8df15",
+        "risk": {
+          "calculated_score_norm": 23,
+          "calculated_level": "Low"
+        }
+      },
+      "ingest_timestamp": "2022-08-15T16:32:16.142561766Z",
+      "@timestamp": "2022-08-12T14:45:36.171Z"
+    },
+    "type": "_doc"
+  }
+}
+
+{
+  "type": "doc",
+  "value": {
+    "id": "4",
+    "index": "risk-score.risk-score-latest-default",
+    "source": {
+      "host": {
+        "name": "zeek-sensor-amsterdam",
+        "risk": {
+          "calculated_score_norm": 70,
+          "calculated_level": "Critical"
+        }
+      },
+      "ingest_timestamp": "2022-08-15T16:32:16.142561766Z",
+      "@timestamp": "2022-08-12T14:45:36.171Z"
+    },
+    "type": "_doc"
+  }
+}
+
+{
+  "type": "doc",
+  "value": {
+    "id": "5",
+    "index": "risk-score.risk-score-latest-default",
+    "source": {
+      "host": {
+        "name": "mothra",
+        "risk": {
+          "calculated_score_norm": 1,
+          "calculated_level": "Low"
+        }
+      },
+      "ingest_timestamp": "2022-08-15T16:32:16.142561766Z",
+      "@timestamp": "2022-08-12T14:45:36.171Z"
+    },
+    "type": "_doc"
+  }
+}
+
+{
+  "type": "doc",
+  "value": {
+    "id": "6",
+    "index": "risk-score.risk-score-latest-default",
+    "source": {
+      "host": {
+        "name": "host-0",
+        "risk": {
+          "calculated_score_norm": 1,
+          "calculated_level": "Low"
+        }
+      },
+      "ingest_timestamp": "2022-08-15T16:32:16.142561766Z",
+      "@timestamp": "2022-08-12T14:45:36.171Z"
+    },
+    "type": "_doc"
+  }
+}
+
+{
+  "type": "doc",
+  "value": {
+    "index": "risk-score.risk-score-latest-default",
+    "id": "7",
+    "source": {
+      "user": {
+        "name": "root",
+        "risk": {
+          "calculated_score_norm": 11,
```

Comment:
Why does this data only have a subset of the ECS fields? Better to add them all to make this more realistic/reusable, no?

Reply:
Same for the mappings.

Reply:
fixed